### PR TITLE
Add GraphQL unit test example for DML queries

### PR DESCRIPTION
**Pull request by olim7t**

### CODE REVIEW

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/QueryFetcherTest.java  1
- graphqlapi/src/test/java/io/stargate/graphql/schema/DmlTestBase.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6309125&bid=20514959)

File: graphqlapi/src/test/java/io/stargate/graphql/schema/GraphQlTestBase.java [hunk 1]
```diff
@@ -64,18 +70,51 @@ public void setupEnvironment() {
...
     }
 
+    // Make every query return an empty result set.
+    // At the time of writing, this is enough to execute all of our fetchers without failure. It
+    // might have to evolve in the future if queries start checking particular elements in the
+    // response (like the '[applied]' column for LWTs).
+    //
+    // As a consequence, our unit tests do not cover the conversion of CQL results back to GraphQL.
+    // We would have to mock the entire result API (ResultSet, Row, Column, ColumnType...), which is
+    // a bit overkill and brittle. The integration tests in the 'testing' module fill that gap.
```
Comment (olim7t) [inline graphqlapi/src/test/java/io/stargate/graphql/schema/GraphQlTestBase.java:80]: This. I think mocking results is not worth the investment.

**Review by dougwettlaufer — DISMISSED** [review]

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- graphqlapi/src/test/java/io/stargate/graphql/schema/DmlTestBase.java  1
- graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/QueryFetcherTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6309125&bid=20514959)

**Review by dougwettlaufer — APPROVED** [review]

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- graphqlapi/src/test/java/io/stargate/graphql/schema/DmlTestBase.java  1
- graphqlapi/src/test/java/io/stargate/graphql/schema/fetchers/dml/QueryFetcherTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6309125&bid=20514959)

**Pull request merged**